src/providers/FusionAuthProvider.tsx

```diff
@@ -54,12 +54,10 @@ export interface FusionAuthConfig extends PropsWithChildren {
 export const FusionAuthProvider: React.FC<FusionAuthConfig> = props => {
     const { children, onRedirectSuccess, onRedirectFail, mePath } = props;
 
-    const [isAuthenticated, setIsAuthenticated] = useState(
-        () => !!Cookies.get('app.at_exp'), // TODO - look at idToken?
-    );
     const [isLoading, setIsLoading] = useState(false);
 
     const [user, setUser] = useState<Record<string, any>>({});
+    const isAuthenticated = useMemo(() => Object.keys(user).length > 0, [user]);
```

Comment:
@JakeLo123 

Is this a robust enough check? `user` could be set to any object and it'd pass this test. Maybe we ought to check for data structure conformance.

Right now, an object with just `{ foo: true }` would pass this test. Maybe one day some one will accidentally set `user` to some arbitrary object by accident and `isAuthenticated` will still be true.

Are there any properties in `user` that are important and likely unchanging?

```json
{
    "applicationId": "some-value",
    "birthdate": "1985-11-23",
    "email": "richard@example.com",
    "email_verified": true,
    "family_name": "Hendricks",
    "given_name": "Richard",
    "roles": [],
    "scope": "some-value",
    "sid": "some-value",
    "sub": "some-value",
    "tid": "some-value"
}
```

This isn't a necessarily blocking comment, but maybe we can start a discussion around this.

Reply:
I agree. This issue address that #77.